de.tubs.variantsync.core/src/de/tubs/variantsync/core/nature/Variant.java

```diff
@@ -9,32 +9,32 @@
 	public static final String NATURE_ID = "de.tubs.variantsync.core.variant";
 	private IProject project;
 
-    @Override
-    public void configure() throws CoreException {
-        // only called once the nature has been set
-    	
-    	//TODO: Create .variantHistory file
-
-        // configure the project...
-    }
-
-    @Override
-    public void deconfigure() throws CoreException {
-        // only called once the nature has been set
-    	
-    	//This should !not! delete the .variantHistory file
-
-        // reset the project configuration...
-    }
-
-    @Override
-    public IProject getProject() {
-        return project;
-    }
-
-    @Override
-    public void setProject(IProject project) {
-        this.project = project;
-    }
+	@Override
+	public void configure() throws CoreException {
+		// only called once the nature has been set
+
+		// TODO: Create .variantHistory file
```

Comment:
Why is this TODO not addressed?

Reply:
Please open an issue and document this gap.

Reply:
This problem will be addressed by the following issue https://github.com/tthuem/VariantSync/issues/57